cli.php

```diff
@@ -11,6 +11,10 @@ class WPSuperCache_Command extends WP_CLI_Command {
 	 * @synopsis [--post_id=<post-id>] [--permalink=<permalink>]
 	 */
 	function flush( $args = array(), $assoc_args = array() ) {
+		require_once( WPCACHEHOME . '/wp-cache-phase1.php' );
+		global $WPSC_HTTP_HOST;
+		$WPSC_HTTP_HOST = parse_url(get_site_url())['host'];
```

Comment:
> On a related note, just wondering why it makes sense

If you have a subdir install and `wp-cli.yml` with `path` in it sits above docroot then you don't have to `cd` to that subdir.
On a WPML multi-domain multilingual site it comes handy to set `url`.
Etc.